apps/newsletters-api/src/app/responses.ts

```diff
@@ -0,0 +1,15 @@
+import type { ApiResponse } from '@newsletters-nx/newsletters-data-client';
+
+export const makeErrorResponse = (message: string): ApiResponse<unknown> => ({
+	ok: false,
+	message,
+	results: undefined,
+});
+
+export function makeSuccessResponse<T extends object>(data: T): ApiResponse<T> {
+	return {
+		ok: true,
+		total: Array.isArray(data) ? data.length : undefined,
+		results: data,
```

Comment:
I can see the case for that - so with the /get/:id route, the results should be an array of one object, rather than just the object itself?

If we go for that, we should change the type definition so the we can put `ApiResponse<Newsletter>` rather than `ApiResponse<Newsletter[]>`